aten/src/ATen/native/Normalization.cpp

```diff
@@ -508,7 +525,30 @@ std::tuple<Tensor, Tensor, Tensor> _batch_norm_impl_index_backward(
   const Tensor& save_mean = c10::value_or_else(save_mean_opt, [] {return Tensor();});
   const Tensor& save_var_transform = c10::value_or_else(save_var_transform_opt, [] {return Tensor();});
 
-  if (impl_index == 0) {
+  if (input.numel()==0) {
+    int64_t num_batch = input.size(0);
+    int64_t num_feature = input.size(1);
+    std::vector<int64_t> dims{0, 2};
+
+    //don't return view of input, don't return empty tensor because it will break gradient chain
+    auto grad_input = grad_output.clone();
```

Comment:
why do you need this clone? (It's cheap for an empty tensor of course, but still)

Reply:
This was blindly copied from `batch_norm`. 
https://github.com/pytorch/pytorch/pull/57321/files#diff-9e36531b6ea57776251e04cf2e1b84a94145020d870fed3d7f82388c47371483L533

I remember we had issue with `view`, though I think that's only with autodiff, but I was too scared to touch it.
Let me try to change it and see if the world blows up.

Reply:
took another look at this, won't we forwarding `input` with the absence of weight/bias. That's going to be surprising and tricky to debug. I think we should keep it as is.

Reply:
`_batch_norm_impl_index_backward` is a native function with some alias informations associated with it. So we have to respect it.
 In particular, that function declare that the computed gradient doesn't alias the grad_output. So it shouldn't